python/test/unit/tools/test_aot.py

```diff
@@ -435,3 +435,20 @@ def test_ttgir_to_ptx():
         ptx = k.asm["ptx"]
         assert ".target sm_80" in ptx
         assert ".address_size 64" in ptx
+
+
+def test_ttgir_to_hasco():
```

Comment:
Don't think this is added to the CI yet? You can check [here](https://github.com/triton-lang/triton/blob/4d791f09355ba4e82123e72af29958d588092b3a/.github/workflows/integration-tests-amd.yml#L104). (Also probably need to disable some of the above tests with `is_hip` check.